README.md

```diff
@@ -1,128 +1,207 @@
+In any supported game (**except Touhou 9**), you can press the **backspace** key to enter the backspace menu; the backspace menu will show you a variety of options that can be activated using the function (F) keys.
 
-NOTE for Steam Deck users: these options are also **clickable**. This means that you can use your left thumb to tap the options to toggle them. Therefore, you only need to map backspace in Steam Input.
+Do note that on the **Steam Deck** the options are clickable; you can use your left thumb to tap the options to toggle them. As a result, you only need to map the backspace key in Steam Input.
 
-![TH18](https://user-images.githubusercontent.com/23106652/174434813-73748a66-0f6d-4c6e-9f3a-895a49b93434.png)
-![TH17](https://user-images.githubusercontent.com/23106652/174434834-6bd93104-1ed2-48ae-a440-9d9cb871ea03.png)
+![Unconnected Marketeers](https://user-images.githubusercontent.com/23106652/174434813-73748a66-0f6d-4c6e-9f3a-895a49b93434.png)
+![Wily Beast and Weakest Creature](https://user-images.githubusercontent.com/23106652/174434834-6bd93104-1ed2-48ae-a440-9d9cb871ea03.png)
 
-## Advanced options
-In any supported game, you can press F12 to access the advanced options.
-![TH08](https://user-images.githubusercontent.com/23106652/174434977-683da583-324b-4bd5-8408-13373dfd5a93.png)
-![TH18](https://user-images.githubusercontent.com/23106652/174435006-e906d30d-0ef5-4930-ae57-1f0919beb5af.png)
+## <a name="advanced-options"></a> **Advanced Options**
+In any supported game, you may use the F12 key to access the advanced options; these options may offer fixes or patches related to the game or other quality of life features.
+
+![Imperishable Night](https://user-images.githubusercontent.com/23106652/174434977-683da583-324b-4bd5-8408-13373dfd5a93.png)
+![Unconnected Marketeers](https://user-images.githubusercontent.com/23106652/174435006-e906d30d-0ef5-4930-ae57-1f0919beb5af.png)
+
+## <a name="statistics"></a> **Statistics (WIP)**
+If you are playing Touhou 13, you can press the "L" key to access a more advanced HUD.
 
-## Statistics (WIP)
-In TH13, press L to view a more advanced HUD
 ![TH13](https://user-images.githubusercontent.com/23106652/174435095-51554934-83d3-4d05-847d-834f3068bc28.png)
 
-## Game specific features
-### TH185 Force Wave
-![grafik](https://github.com/touhouworldcup/thprac/assets/23106652/02c55e5e-5c89-462f-beea-9ab07cbb1051)
+## <a name="game-specific-features"></a> **Game-specific Features**
+Some games such as **100th Black Market** and **Phantasmagoria of Flower View** have game specific features to make practicing easier.
+
+### 100th BM - Force Wave
+![100th Black Market](https://github.com/touhouworldcup/thprac/assets/23106652/02c55e5e-5c89-462f-beea-9ab07cbb1051)
 
-If thprac is enabled ("Mode" is set to "Custom"), this popup will appear whenever the game is about to randomly choose a wave.
+If thprac is enabled using "Custom" mode, the popup above will appear when the game is about to choose a wave.
```

Comment:
The reason I phrased it as "Mode" is set to "Custom" is because in this dropdown that's labelled "Mode" the option that the user has to select is called "Custom"
![grafik](https://github.com/user-attachments/assets/5b54bbbe-dc46-45e8-bbc8-e93de98a6e14)